tests/boost/CMakeLists.txt

```diff
@@ -65,6 +65,20 @@ target_link_libraries(dataset_broker_producer2 PRIVATE libexternal kotekan_utils
 target_link_libraries(dataset_broker_producer PRIVATE libexternal kotekan_utils kotekan_core)
 target_link_libraries(dataset_broker_consumer PRIVATE libexternal kotekan_utils kotekan_core)
 
+# Many tests now need the Cuda include directory for float16_t via visUtil.hpp
```

Comment:
Will this build if `cuda` isn't enabled/installed, but boost tests are?   Or are we making cuda a requirement of the boost tests?

Reply:
I just tested with `cmake -DUSE_CUDA=OFF ..` and these tests are still built and they run.  So I think it's fine